### ISSUE TITLE
[BUG] After login if we press back button then Confirm form Resubmission page shows.

### ISSUE THREAD

**Issue by gurjyot** [q2a/question2answer#548]
This is happening if, user login through the login page. Please fix this, it's very annoying. I have tested some other platforms but they just redirect the user to homepage instead of going back to the login page. So shouldn't it be the same for Q2A?

I was also searching how to fix this and maybe these links can help.

https://stackoverflow.com/questions/15226744
https://stackoverflow.com/questions/32897673

**Comment by svivian** [comment]
Yes, this has been an annoyance of mine for a while too. It happens in several places such as the admin settings. The "Post/Redirect/Get pattern" as mentioned in that SO question is the standard on the web these days. I'm making some changes to the structure of the "pages" in Q2A (using a Controller type structure) so I'll likely tackle some of these issues.

**Comment by gurjyot** [comment]
Can this issue be addressed in Q2A 1.8 release only. It's kind of difficult to run a professional website with such a bug and expect users to just ignore this issue. 
It would be too late till 1.9 and getting a bug fixed is essential for any new version.
